Updated Identifier breakdown information (#4263)
segmentio/segment-docs: src/engage/audiences/index.md

    (If the audience recalculates between when you click Generate and when you download the file, you might want to regenerate the file. The CSV is a snapshot from when you clicked Generate, and could be outdated.)</td>
  </tr>
</table>

## Identifier Breakdown

> The audience summary is a breakdown of the percentages of external_ids of users in the audience. These are the default ID's that have been included in Identity resolution configuration. It shows the portion of the audience that has each identifier, and is used to verify the audience size/profiles are correct and as expected.

Reviewer: ```suggestion
The audience summary is a breakdown of the percentages of external_ids of users in the audience. These are the default IDs that Segment includes in your Identity resolution configuration. Segment displays the percentage of the audience with each identifier, which you can use to verify the audience size and profiles are correct.
```

> The audience summary is a breakdown of the percentages of external_ids of users in the audience. These are the default IDs that Segment includes in your Identity resolution configuration. Segment displays the percentage of the audience with each identifier, which you can use to verify the audience size and profiles are correct.

Reviewer: ```suggestion
The audience summary is a breakdown of the percentages of external_ids of users in the audience. These are the default IDs that Segment includes in the Identity resolution configuration. Segment displays the percentage of the audience with each identifier, which you can use to verify the audience size and profiles are correct.
```

> info ""

> > The Identifier breakdown won't show custom ID's included in identity resolution configuration. Only default external IDs will be shown in the breakdown.

Reviewer: ```suggestion
> The Identifier Breakdown won't show custom IDs included in your Identity resolution configuration. Segment only displays external IDs in the breakdown.
```

Reviewer: UI displays `Identifier Breakdown`.
